inc/class-shortcode-ui.php

```diff
@@ -366,38 +394,63 @@ private function render_shortcode_for_preview( $shortcode, $post_id = null ) {
 	}
 
 	/**
-	 * Get a bunch of shortcodes to render in MCE preview.
+	 * Get a preview for a single shortcode to render in MCE preview.
 	 */
-	public function handle_ajax_bulk_do_shortcode() {
-
-		if ( is_array( $_POST['queries'] ) ) {
-
-			$responses = array();
+	public function handle_shortcode_preview( WP_REST_Request $request ) {
+		return $this->get_shortcode_preview( $request->get_param( 'query' ) );
```

Comment:
I'm not sure what this function is doing, but we don't currently have a `get_shortcode_preview()` method here...

I think this functionality might be vestigial anyways, currently all previews are bulk requests even if they're only for one shortcode.